readme.txt

```diff
@@ -3,4 +3,4 @@ Gleam-Wordpress
 
 Widget shortcode example:
 
-[gleam_widget url="https://gleam.io/iR0GQ/gleam-demo-competition"]Gleam Demo Competition[/gleam_widget]
+[gleam url="https://gleam.io/iR0GQ/gleam-demo-competition"]Gleam Demo Competition[/gleam]
```

Comment:
**Question**
Will there be migration implications for this?  E.g. if people update the plugin then their old shortcodes will break?

Reply:
No, because this code only moves javascript inclusion to footer